packages/core/src/components/tree-item/tree-item.scss

```
@@ -28,9 +29,14 @@
     height: vars.$x-large-space;
     flex-grow: 1;
     align-items: center;
+    outline: none;
 
     @include text-truncation.ellipsis;
 
+    &:focus-visible {
+      @include focus-visible.ix-focus-visible();
```

Stylelint (17.14.0)

[error] Unexpected parentheses in argumentless mixin "focus-visible.ix-focus-visible" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)